Assets/Scripts/Enemies/FireSlime.cs

```diff
@@ -46,6 +46,10 @@ void Move(DIRECTION _first_direction, DIRECTION _final_direction)
     {
         Tile nextTile = m_CurrentTile.GetTile(_first_direction);
         Tile finalTile = nextTile.GetTile(_final_direction);
+        if(Map.instance.IsTileBlocked(finalTile))
+        {
+            return;
+        }
```

Comment:
Chestia asta  nu ii permite inamicului sa sape. Ori adaug o codificare pt spatiu ocupat de inamic ori schimb sistemul. 
O sa ramana in pull request varianta asta.
De facut mai tarziu.